src/Components/CardboardMultiSelect/CardboardMultiSelect.styles.ts

```diff
@@ -0,0 +1,101 @@
+import { FontSizes, Theme } from '@fluentui/react';
+import { StylesConfig } from 'react-select';
+import {
+    ICardboardMultiSelectStyleProps,
+    ICardboardMultiSelectStyles
+} from './CardboardMultiSelect.types';
+
+export const classPrefix = 'cb-cardboardmultiselect';
+const classNames = {
+    root: `${classPrefix}-root`,
+    input: `${classPrefix}-input`
+};
+export const getStyles = (
+    _props: ICardboardMultiSelectStyleProps
+): ICardboardMultiSelectStyles => {
+    return {
+        root: [classNames.root],
+        input: [classNames.input],
+        subComponentStyles: {}
+    };
+};
+
+export const getMultiSelectStyles = (theme: Theme): StylesConfig => {
```

Comment:
can we not inherit some of our styling from the other react-select control? This feels very duplicative.